dynawo/sources/Models/Modelica/Dynawo/Examples/Nordic/Components/GeneratorWithControl/GeneratorSynchronousThreeWindingsWithControl.mo

```diff
@@ -0,0 +1,96 @@
+    Placement(visible = true, transformation(origin = {-72, 48}, extent = {{-28, -28}, {28, 28}}, rotation = 0)));
+  Controls.Governor governorNordic(sigma = GeneratorParameters.govParamValues[gen,GeneratorParameters.govParams.sigma], kI = GeneratorParameters.govParamValues[gen,GeneratorParameters.govParams.GovKi], kP = GeneratorParameters.govParamValues[gen,GeneratorParameters.govParams.GovKp]) annotation(
+    Placement(visible = true, transformation(origin = {-73, -33}, extent = {{-27, -27}, {27, 27}}, rotation = 0)));
+  Connectors.ACPower terminal annotation(
+    Placement(visible = true, transformation(origin = {-3.55271e-15, 104}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {0, 3.55271e-15}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  // Different bases for PGenPu, ifPu and efdPu
+  Blocks.Math.Gain BaseChangeSnRef2PNom(k = SystemBase.SnRef / PNom_BaseChange) annotation(
+    Placement(visible = true, transformation(origin = {-139, -49}, extent = {{-17, -17}, {17, 17}}, rotation = 0)));
+  Blocks.Math.Gain Reciprocal2NonReciprocalPUSystem(k = generatorSynchronous.MdPPu) annotation(
+    Placement(visible = true, transformation(origin = {-139, 65}, extent = {{-17, -17}, {17, 17}}, rotation = 0)));
+protected
+  parameter Real PNom_BaseChange = if not GeneratorParameters.genParamValues[gen, GeneratorParameters.genParams.PNom] == 0 then GeneratorParameters.genParamValues[gen, GeneratorParameters.genParams.PNom] else 1;
+  parameter Types.ActivePowerPu P0Pu;
+  parameter Types.ReactivePowerPu Q0Pu;
+  parameter Types.VoltageModulePu U0Pu;
+  parameter Types.Angle UPhase0;
+initial algorithm
+  governorNordic.Pm0Pu := generatorSynchronous.Pm0Pu;
+  ExciterAvrPssOel.Efd0Pu := generatorSynchronous.Efd0Pu;
+  ExciterAvrPssOel.UStator0Pu := generatorSynchronous.UStator0Pu;
+  ExciterAvrPssOel.If0Pu := generatorSynchronous.If0Pu;
+/*initial equation
```

Comment:
If not needed they can be deleted

Reply:
resolved, see [d146726](https://github.com/dynawo/dynawo/pull/2394/commits/d14672683da0d677556a70ce6fab3936ca672551)